godtools/App/Features/Dashboard/Domain/DashboardTabType.swift

```diff
@@ -1,16 +1,15 @@
 //
-//  ToolsMenuPageType.swift
+//  DashboardTabType.swift
```

Comment:
Could this be named to ```DashboardTabTypeDomainModel```.   Just to help us see that this is defined in our domain layer.